apps/onboarding/src/session/session.entity.ts

```diff
@@ -0,0 +1,48 @@
+import {Column, Entity, PrimaryGeneratedColumn} from "typeorm"
+
+@Entity()
+export class Session {
+
+    public static of(params: Partial<Session>): Session {
+        const session = new Session()
+    
+        Object.assign(session, params)
+    
+        return session
+      }
+
+    @PrimaryGeneratedColumn('uuid')
+    id: string
+
+    @Column()
+    externalAccount: string
+
+    @Column()
+    requestedAttestations: number
+
+    @Column()
+    completedAttestations: number
+
+    @Column('simple-json')
+    meta?: object
+
+    @Column('date')
+    createdAt: string
+
+    @Column('date')
+    expiredAt: string
+
+    @Column('date')
+    completedAt: string
+}
+    
+export class SessionRepositoryFake {
```

Comment:
Let's move this to the test if it's for testing. 
Also checkout the answer here: https://stackoverflow.com/questions/57099863/spyon-typeorm-repository-to-change-the-return-value-for-unit-testing-nestjs

For an approach on how to handle repository mocking in testing.